docs/docs/uploads-legacy.md

```
@@ -0,0 +1,947 @@
+```
+
+The `$extends` method is used to extend the functionality of your Prisma client
+by adding
+
+- [Query extensions](https://www.prisma.io/docs/orm/prisma-client/client-extensions/query)
+  which will intercept your `create`, `update`, `delete` operations <br/>
+- [Result extensions](https://www.prisma.io/docs/orm/prisma-client/client-extensions/result)
+  for your stored files - which gives you helper methods on the result of your
+  prisma query
+
+More details on these extensions can be found [here](#storage-prisma-extension).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)